flume/conf/README.md

```diff
@@ -4,26 +4,26 @@ From version 0.6.0 Cygnus is able to start multiple instances by renaming and co
 templates files of this directory.
 
 There are two types of configuration template files:
-1- Those applying to all the Cygnus instances; thus, they are unique.
-2- Those applying specifically to one Cygnus instance; thus, there is a configuration file per each instance.
 
-## 1- Unique files for all instances
+* Those applying to all the Cygnus instances; thus, they are unique.
+* Those applying specifically to one Cygnus instance; thus, there is a configuration file per each instance.
 
-For them is necessary rename (or better copy) it without `.template` extension.
-These files are: `log4j.properties.template`, `krb5.conf.template` and `matching_table.conf.template`
+## Unique files for all instances
+
+For them is necessary rename (or better copy) it without `.template` extension. These files are: `log4j.properties.template`, `flume-env.sh.template`, `krb5.conf.template` and `matching_table.conf.template`
 
 ```bash
 cp /usr/cygnus/conf/log4j.properties.template /usr/cygnus/conf/log4j.properties
+cp /usr/cygnus/conf/flume-env.sh.template /usr/cygnus/conf/flume-env.sh
 cp /usr/cygnus/conf/krb5.conf.template /usr/cygnus/conf/krb5.conf
 cp /usr/cygnus/conf/matching_table.conf.template /usr/cygnus/conf/matching_table.conf
 ```
 
 Then each file can be edited in order to change the configuration that will affect to all intances of Cygnus.
 
-## 2- Files for each instance
+## Files for each instance
 
-The other files are `cygnus.conf.template` and `flume.conf.template`. These files should be copied 
-for each intantance of Cygnus that want to run.
+The other files are `cygnus.conf.template` and `flume.conf.template`. These files should be copied for each intantance of Cygnus that want to run.
```

Comment:
Fixed in commit https://github.com/telefonicaid/fiware-connectors/commit/f5e6a3201684bd03c9276ef30df68b6a4f89079b